src/publisher/api_v2_views.py

```diff
@@ -18,6 +16,12 @@
 LOG = logging.getLogger(__name__)
 
 
+class HttpResponse(DjangoHttpResponse):
+    @property
+    def content_type(self):
+        return self.get("content-type", None)
```

Comment:
test. does 'content-type' also cover 'Content-Type' ?

Reply:
yes, http headers are case-insensitive. and `application/json` doesn't need to specify a `charset` because it's utf-8, which isn't a charset but an encoding ...